force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        If(a != null && b != null){ //Verifing if either of the variables is null
+            Integer sum = a + b; //calculating the sum of two variables
+            return sum; //Returned the sum value
+        }
+        else 
+        
+        {return null;} // If the above condition is false, the code will return null as the output
```

Comment:
Update your spacing and indentations 
```suggestion
        {
          return null;
        } // If the above condition is false, the code will return null as the output
```

Reply:
Sure, I'll follow this. I had this misconception of having as less number of lines as possible. So tried keeping it short. Does having more lines make a code compile slowly?